apps/site/lib/site_web/templates/page/_tabbed_nav.html.eex

```diff
@@ -0,0 +1,33 @@
+<div class="m-tabbed-nav">
+  <div class="m-tabbed-nav__items">
+    <div class="m-tabbed-nav__item-container">
+      <div class="m-tabbed-nav__item active" data-tab-type="schedules">
+        <span class="m-tabbed-nav__icon notranslate">
+          <%= svg("icon-map-default.svg") %>
+        </span>
+        <span class="m-tabbed-nav__icon-text">Schedules</span>
+      </div>
+    </div>
+    <div class="m-tabbed-nav__item-container">
+      <div class="m-tabbed-nav__item" data-tab-type="trip-planner">
+        <span class="m-tabbed-nav__icon notranslate">
+          <%= svg("icon-trip-planner-default.svg") %>
+        </span>
+        <a href="/trip-planner" class="m-tabbed-nav__icon-text">Trip Planner</a>
+      </div>
+    </div>
+    <div class="m-tabbed-nav__item-container">
+      <div class="m-tabbed-nav__item" data-tab-type="alerts">
+        <span class="m-tabbed-nav__icon notranslate">
```

Comment:
You can omit `notranslate` class here -- the `svg` function below adds it. As it's written right now this is creating:

```
<span class="m-tabbed-nav__icon notranslate">
  <span class="notranslate c-svg__icon-map-default">
      // svg content cut for brevity
  </span>
</span>
```

Optionally you can remove this `<span>` wrapper altogether and style it using a selector like `.m-tabbed-nav__item [class*='c-svg__icon']`